### PR TITLE
fix(rome_playground): Unicode Support

### PR DESCRIPTION
**Pull request by MichaReiser**
They playground crashed because `btoa` only supports ASCII.
This PR ensures that non ASCII characters are propertly encoded before running the code throuh `btoa`.

## Test Plan

Screenshot from 2022-03-31 11-38-50

### CODE REVIEW

File: .github/workflows/deploy_playground.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
           node-version: '14'
       - run: npm install --prefix crates/rome_playground
       - uses: jetli/wasm-pack-action@v0.3.0
-      - run: wasm-pack build crates/rome_playground --target web
+      - run: wasm-pack build crates/rome_playground --target web --release
```
Comment (MichaReiser) [inline .github/workflows/deploy_playground.yml:25]: This reduces the wasm size by 0.5 mb

File: crates/rome_playground/src/App.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import CodeEditor from "@uiw/react-textarea-code-editor";
 import "react-tabs/style/react-tabs.css";
 import init, { run } from "../pkg/rome_playground";
 import { Tabs, Tab, TabList, TabPanel } from "react-tabs";
-import {useEffect, useMemo, useState} from "react";
+import { useEffect, useMemo, useState } from "react";
```
Comment (MichaReiser) [inline crates/rome_playground/src/App.tsx:5]: Nit: We should probably move the playground into the website directory. It isn't a proper "rome" rust crate

File: .github/workflows/deploy_playground.yml (hunk 1 above)
Comment (xunilrj) [reply .github/workflows/deploy_playground.yml:25]: hum... if we have not done anything to reduce, can this help?
https://rustwasm.github.io/book/reference/code-size.html

**Review by xunilrj — APPROVED** [review]

File: crates/rome_playground/src/App.tsx (hunk 2 above)
Comment (ematipico) [reply crates/rome_playground/src/App.tsx:5]: It's the same thing that I suggested a while ago.

**Review by ematipico — APPROVED** [review]

**Pull request merged**